src/Compilers/CSharp/Portable/Lowering/Instrumentation/DebugInfoInjector.cs

```diff
@@ -352,7 +352,7 @@ public override BoundStatement InstrumentPatternSwitchStatement(BoundPatternSwit
             SwitchStatementSyntax switchSyntax = (SwitchStatementSyntax)original.Syntax;
             TextSpan switchSequencePointSpan = TextSpan.FromBounds(
                 switchSyntax.SwitchKeyword.SpanStart,
-                switchSyntax.CloseParenToken.Span.End);
+                (switchSyntax.CloseParenToken != default) ? switchSyntax.CloseParenToken.Span.End : switchSyntax.Expression.Span.End);
```

Comment:
>switchSyntax.CloseParenToken != default [](start = 17, length = 39)

`.IsMissing`?

Reply:
That would return the wrong (`false`) result. A "missing" token is one that was inserted by the parser. There is no token here at all.

---
In reply to: [179895304](https://github.com/dotnet/roslyn/pull/25963#discussion_r179895304) [](ancestors = 179895304)